plugin/cmake4vim.vim

```diff
@@ -63,6 +63,8 @@ if !exists('g:cmake_vimspector_support')
     let g:cmake_vimspector_support = 0
 endif
 
+let g:cmake_variants = get( g:, 'cmake_variants', {} )
```

Comment:
Hmm, I think we can use this approach for all options. We can do it in the separate PR.